apps/emqx_ft/test/emqx_ft_conf_SUITE.erl

```diff
@@ -34,7 +34,9 @@ end_per_suite(_Config) ->
 init_per_testcase(_Case, Config) ->
     % NOTE: running each testcase with clean config
     _ = emqx_config:save_schema_mod_and_names(emqx_ft_schema),
-    ok = emqx_common_test_helpers:start_apps([emqx_conf, emqx_ft], fun(_) -> ok end),
+    ok = emqx_common_test_helpers:start_apps(
+        [emqx_conf, emqx_ft], emqx_ft_test_helpers:env_handler(Config)
+    ),
```

Comment:
This will change the meaning of the successful test passes here. Before they were specifically made to run without any initial `emqx_ft` configuration (see NOTE above), to show how things work when someone enables `emqx_ft` at runtime. I'd vote at least for keeping that in some form, say, in a separate test group.

Reply:
Does starting with an empty env configuration handler model an existing situation? The app cannot work without `emqx_conf` (because only emqx_ft schema is injected into it), and when started with `emqx_conf`, the config will be filled either with default or provided schema.

One of the ways to achieve no initial configuration is to 1) stop the app 2) delete its config _explicitly_ 3) try to start

I don't think we should handle such a situation 😅 
I prefer not to handle "empty" config and fail explicitly, because this is a situation we shouldn't normally reach, and it is a bug if we reach it 🤔

Reply:
> Does starting with an empty env configuration handler model an existing situation?

Your answer suggests that this would model situation _emqx starts with default configuration_. Which on first look sounds testcase-worthy.

> this is a situation we shouldn't normally reach

I agree with the general idea. 👍🏼 But still wondering what we'll see if the user will put `file_transfer = null` in the config file.

Reply:
>this would model situation emqx starts with default configuration
Maybe this is the fault of our test helpers but this is not the case: starting with `emqx_common_test_helpers:start_apps` and empty handler _does not_ model starting EMQX with empty config because `emqx_common_test_helpers:start_apps` does not make population with default configs.

When we start EMQX without `file_transfer` at all or with `file_transfer = null` population with default values happens. So instead, some `env_handler` is needed to emulate this initial default values.